org.greenfoot.Greenfoot.appdata.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>org.greenfoot.Greenfoot</id>
+  <name>Greenfoot</name>
+  <developer_name>King's College London</developer_name>
+  <summary>Greenfoot</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0-with-classpath-exception</project_license>
+  <url type="homepage">https://greenfoot.org</url>
+  <description><p>Greenfoot is an integrated development environment using Java or Stride designed primarily for educational purposes at the high school and undergraduate level. It allows easy development of two-dimensional graphical applications, such as simulations and interactive games.</p></description>
+  <screenshots>
+    <screenshot type="default">
+      <caption>Editor Window</caption>
+      <image>https://raw.githubusercontent.com/flathub/org.greenfoot.Greenfoot/master/img/02.png</image>
+    </screenshot>
+    <screenshot>
+      <caption>Main Window</caption>
+      <image>https://raw.githubusercontent.com/flathub/org.greenfoot.Greenfoot/master/img/01.png</image>
+    </screenshot>
+    <screenshot>
+      <caption>Property Viewer</caption>
+      <image>https://raw.githubusercontent.com/flathub/org.greenfoot.Greenfoot/master/img/03.png</image>
+    </screenshot>
+  </screenshots>
+  <categories>
+    <category>Development</category>
+    <category>Education</category>
+  </categories>
+  <icon type="remote" height="256" width="256">https://raw.githubusercontent.com/flathub/org.greenfoot.Greenfoot/master/img/greenfoot-256.png</icon>
+  <icon type="remote" height="128" width="128">https://raw.githubusercontent.com/flathub/org.greenfoot.Greenfoot/master/img/greenfoot-128.png</icon>
+  <icon type="remote" height="64" width="64">https://raw.githubusercontent.com/flathub/org.greenfoot.Greenfoot/master/img/greenfoot-64.png</icon>
```

Comment:
```suggestion
```

This should be detected automatically by the tooling, as you already ship icons with the app.